test/components/AppTest.js

```diff
@@ -39,6 +39,12 @@ describe('App', () => {
       assert(ReactDOM.findDOMNode(appComponent))
     })
 
+    it('has a tabbed nav', () => {
+      const appComponent = renderPage(store)
+      const app = ReactDOM.findDOMNode(appComponent)
+      assert(app.querySelectorAll('tabbed-nav'))
```

Comment:
This doesn't look right. I think you mean `assert(app.querySelector('.tabbed-nav'))` or even `assert.equal(1, app.querySelectorAll('.tabbed-nav').length)`.